internal/store/postgres/domain.go

```diff
@@ -0,0 +1,30 @@
+package postgres
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/raystack/frontier/core/domain"
+)
+
+type Domain struct {
+	ID        string       `db:"id"`
+	OrgID     string       `db:"org_id"`
+	Name      string       `db:"name"`
+	Token     string       `db:"token"`
+	State     string       `db:"state"`
+	UpdatedAt sql.NullTime `db:"updated_at"`
```

Comment:
created and updated will be same when created for the first time so it should not be null right?